README.md

```diff
@@ -144,27 +144,7 @@ Goal is to  get gonomics into the “src” directory of go. This path needs to
 
 <br>
 
-*Workspace set up for basic gonomics installation (most users):*
-<ul>
-<ul>
-
-    cd $GOPATH/src
-    go get
-</ul>
-this should install gonomics and all its dependencies.
-<ul>
-
-    cd /github.com/vertgenlab/gonomics/ 
-    go test ./…
-</ul>
-This should print to screen a line for each test with ok printed in the left margin when something passes
-</ul>
-</ul>
-<br>
-<br>
-<br>
-
-*Gonomics with version control (for contributors of gonomics):*
+*Gonomics installation with version control (for both users and contributors of gonomics):*
 <ul>
 
     cd $GOPATH/src
```

Comment:
Go also modified the way that GOPATH works. I believe it is no longer required that code be inside GOPATH so I think we could delete the entire section about setting GOBIN, GOROOT, and GOPATH and tell the user to just run `git clone` wherever they like. Someone should double check that though before changing. May be something for a future PR.

Reply:
I think we can do this, because when I was walking through installing go and gonomics with Evon on 2022-04-14, we didn't do the GOBIN, GOROOT, GOPATH stuff. We just downloaded go from the website, and I had Evon mkdir GOPATH/src/github.com/vertgenlab/gonomics and do git clone. I will make some changes and others can comment on if they are approprite or not